src/library/objects/Messengers.js

```diff
@@ -24,7 +24,7 @@ Sample Usage:
 (function(){
 	"use strict";
 	
-	console.info("KC3改 Messengers loaded");
+	console.log("KC3改 Messengers loaded");

```

Comment:
About this console logging... from chromium 55+, new extension box named 'collect errors' introduced, and enabled by default for unpacked extension. I guess this is the motivation u change it. But it is still collected as errors even changed to `.log` 😭  I think we may not care about this, because packed ext. and developers could both disable the 'collect errors' to avoid errors icon appear...